src/15_classes.py

```diff
@@ -2,20 +2,50 @@
 # constructor
 
 # YOUR CODE HERE
+class LatLon:
+  lat = 0
+  lon = 0
+  def __init__(self, lat, lon):
+    self.lat = lat
+    self.lon = lon
 
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        self.name = name
 
+        # Invoke __init__ of parent class
+        LatLon.__init__(self, lat, lon)
```

Comment:
You could also call super().__init__(self, lat, lon) instead of  LatLon.__init__(self, lat, lon)